src/main/scala/com/trueconnectivity/e2e/E2eTestPlugin.scala

```diff
@@ -23,34 +23,32 @@ object E2eTestPlugin extends AutoPlugin {
     import DockerComposePlugin.autoImport._
 
     lazy val settings: Seq[Setting[_]] = Seq(
-      testCasesJar := (IntegrationTest / packageBin / artifactPath)
-        .value
-        .getAbsolutePath,
+      testCasesJar := (IntegrationTest / packageBin / artifactPath).value.getAbsolutePath,
       testCasesPackageTask := {
         HealthCheck.waitUntilHealthy(
```

Comment:
I wonder why we even need that extra task. Can we not do the healthcheck directly in the compose file?(https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck)

But maybe not worth the effort to change if it works as is.

Reply:
i dont think that actually stops the docker compose command from completing, its just used to set the state of a container (starting, healthy, unhealthy)

Reply:
Right, we could maybe check the status and wait directly inside sbt-docker-compose (with filtering the container ids by health as well). Then we wouldn't need to assign and read the ports to env vars at all. But just an idea.